package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@amcharts/amcharts4": "^4.8.5",
```

Comment:
how did you chose this library ? have you checked the licensing model ?

Reply:
Alexis used it on the other timeline repository.
There is a free-to-use version of amcharts, but it imposes to keep the logo on the timeline. 
"Free license : use anywhere you want as long as you don't mind a small amCharts attribution on charts" https://www.amcharts.com/online-store/

I wanted to use other libraries but I didn't find any one really satisfying, the idea was to keep this first test version until finding a better solution (maybe by customizing another open source timeline)